global_costmap:
  global_frame: map
  robot_base_frame: base_link

  update_frequency: 5.0
  publish_frequency: 5.0
  transform_tolerance: 0.5
  

  static_layer:
    enabled: true
    map_topic: /map
    subscribe_to_updates: false

  plugins: 
    - name: static_layer
      type: "costmap_2d::StaticLayer"
    - name: obstacle_layer
      type: "costmap_2d::ObstacleLayer"
    - name: inflation_layer
      type: "costmap_2d::InflationLayer"
